cluster-operator/src/main/java/io/strimzi/operator/cluster/model/AbstractModel.java

```diff
@@ -58,6 +62,8 @@
 
     protected static final Logger log = LogManager.getLogger(AbstractModel.class.getName());
 
+    protected static final int DEFAULT_CERTS_EXPIRATION_DAYS = 365;
```

Comment:
We currently do not support any changes to this. So it should not be called `DEFAULT_CERTS_EXPIRATION_DAYS` but just  `CERTS_EXPIRATION_DAYS`? When I see `DEFAULT_` I normally expect that the actual used value would be somewhere else.

Reply:
Yeah agree on that.